ibis/expr/types.py

```diff
@@ -192,14 +194,24 @@ def execute(self, limit='default', params=None, **kwargs):
           Pass an integer to effect a specific row limit. limit=None means "no
           limit". The default is whatever is in ibis.options.
 
+        timecontext: Tuple(pd.Timestamp, pd.Timestamp) or None, default None
+           Defines a time range of [begin, end). If table contains a time
```

Comment:
Defines a time range of [begin, end). When defined, the execution will only compute result for data inside the time range. This is conceptually same as a time filter.

Reply:
Thanks, I re-phrased the user doc.